aws/resource_aws_dynamodb_table.go

```diff
@@ -83,13 +83,22 @@ func resourceAwsDynamoDbTable() *schema.Resource {
 				Optional: true,
 				ForceNew: true,
 			},
+			"billing_mode": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Default:  dynamodb.BillingModeProvisioned,
+				ValidateFunc: validation.StringInSlice([]string{
+					dynamodb.BillingModePayPerRequest,
+					dynamodb.BillingModeProvisioned,
+				}, false),
+			},
 			"write_capacity": {
 				Type:     schema.TypeInt,
-				Required: true,
+				Required: false,
```

Comment:
Instead of `Required: false`, these should be updated to `Optional: true` 👍 

```suggestion
				Optional: true,
```